Aliucord/src/main/java/com/aliucord/widgets/UpdaterPluginCard.java

```diff
@@ -43,6 +39,7 @@ public UpdaterPluginCard(Context context, String plugin, Runnable forceUpdate) {
         setContentPadding(padding, padding, padding, padding);
 
         Plugin p = PluginManager.plugins.get(plugin);
+        assert p != null;
```

Comment:
maybe this should be changed to a more appropriate error? but it should never happen anyway so whatever